pymysql/cursors.py

```diff
@@ -145,17 +145,18 @@ def executemany(self, query, args):
 
         m = RE_INSERT_VALUES.match(query)
         if m:
-            q_values = m.group(1).rstrip()
+            q_prefix = m.group(1)
+            q_values = m.group(2).rstrip()
+            q_postfix = bytearray(m.group(3) or '')
```

Comment:
prefix is encoded in `_do_execute_many()`. postfix should be encoded in same way.